src/hash.cr

```diff
@@ -1343,6 +1343,15 @@ class Hash(K, V)
     self
   end
 
+  # Adds the contents of *other* to this hash. 
+  # The value of each duplicate key is determined by calling the block with the key, its value in this hash and its value in *other*.
```

Comment:
Maybe a little bit more verbose description, but help understanding:
```suggestion
  # If a key exists in both hashes, the given block is called to determine the value to be used. The block arguments are the key, the value in `self` and the value in *other*.
```

Reply:
You're right i chaged it.